axelrod/moran.py

```diff
@@ -101,6 +119,23 @@ def __init__(self, players, turns=DEFAULT_TURNS, noise=0,
             mutation_targets[key] = [v for (k, v) in sorted(d.items()) if k != key]
         self.mutation_targets = mutation_targets
 
+        if interaction_graph is None:
+            interaction_graph = complete_graph(len(players), loops=False)
+            if reproduction_graph is None:
+                reproduction_graph = complete_graph(len(players))
+        if reproduction_graph is None:
```

Comment:
This is the same default as it was implemented in `MoranProcessGraph` but I'll change it. I've got a further idea for how I'd like to refactor this as well.